contracts/TokenRequest.sol

```diff
@@ -196,12 +197,11 @@ contract TokenRequest is AragonApp {
     *      This function requires the MINT_ROLE permission on the TokenManager specified.
     * @param _tokenRequestId ID of the Token Request
     */
-    function finaliseTokenRequest(uint256 _tokenRequestId) external nonReentrant auth(FINALISE_TOKEN_REQUEST_ROLE) {
-        TokenRequest memory tokenRequestCopy = tokenRequests[_tokenRequestId];
-        require(tokenRequestCopy.requesterAddress != address(0), ERROR_REQUEST_NO_REQUEST);
+    function finaliseTokenRequest(uint256 _tokenRequestId) external nonReentrant auth(FINALISE_TOKEN_REQUEST_ROLE) TokenRequestExists(_tokenRequestId){
```

Comment:
I generally put the `auth()` modifiers last as I find I look for them a lot and they're quicker to spot if always at the end. Up to you though. Also since it's quite long you could put it over multiple lines like this:

```suggestion
    function finaliseTokenRequest(uint256 _tokenRequestId) 
        external 
        nonReentrant 
        TokenRequestExists(_tokenRequestId)
        auth(FINALISE_TOKEN_REQUEST_ROLE) 
    {
```